sources/osgShadow/shaders/shadowReceive.glsl

```diff
@@ -1,4 +1,8 @@
-#pragma include "tapPCF.glsl"
+#ifdef _OUT_DISTANCE
+#define OPT_ARG_outDistance ,out float outDistance
+#else
```

Comment:
might as well use the same `OPT_ARG_INSTANCE_outDistance` trick

Reply:
not sure about the name... maybe OPT_ARG_DECLARATION_stuffs and OPT_ARG_INSTANCE_stuffs
note sure...